src/test/java/uk/gov/ida/eventemitter/EventEmitterConfigurationTest.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.ida.eventemitter;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EventEmitterConfigurationTest {
```

Comment:
Do these tests add anything? 

I think we are just testing a POJO here, which in my view is the kind of trivial object that we shouldn't really be testing. (reasoning being that all code has a maintenance cost, and this test has no realistic chance of catching a bug or preventing a regression, but does make it more awkward to make legitimate changes to the class)

I know there are different views on this kind of thing though, and I'm not trying to change the 'GDS Way', so feel free to disagree with me!

Reply:
I followed TDD to create EventEmitterConfiguration class.

Reply:
Yep - I think I might be outvoted on this one! 🤣 
I'd personally rather not have these tests for the reasons above, but I'm happy to get onboard if that's what the team thinks is best.